doc/examples/animations_pulsating_sphere.py

```diff
@@ -0,0 +1,127 @@
+                                    frames, figsize=(8, 8), **kwargs):
+    """Generate sound particle animation."""
+    velocity = sfs.mono.source.pulsating_sphere_velocity(
+               omega, center, radius, amplitude, grid)
+    displacement = sfs.util.displacement(velocity, omega)
+    phasor = np.exp(1j * 2 * np.pi / frames)
+
+    fig = plt.figure(figsize=figsize)
+    ax = plt.axes()
+    scat = sfs.plot.particles(grid + displacement, **kwargs)
+    ax.axis([grid[0].min(), grid[0].max(), grid[1].min(), grid[1].max()])
+    return animation.FuncAnimation(
+            fig, update_frame_displacement, frames,
+            fargs=(grid, displacement, phasor, scat, frames))
+
+
+def animation_particle_velocity(omega, center, radius, amplitude, grid, frames,
+                                figsize=(8, 8), **kwargs):
+    """Generate particle velocity animation."""
+    velocity = sfs.mono.source.pulsating_sphere_velocity(
+               omega, center, radius, amplitude, grid)
+    phasor = np.exp(1j * 2 * np.pi / frames)
+
+    fig = plt.figure(figsize=figsize)
+    ax = plt.axes()
+    quiv = sfs.plot.vectors(
+            velocity, grid, clim=[-omega * amplitude, omega * amplitude],
+            **kwargs)
+    ax.axis([grid[0].min(), grid[0].max(), grid[1].min(), grid[1].max()])
+    return animation.FuncAnimation(
+            fig, update_frame_velocity, frames,
+            fargs=(velocity, phasor, quiv, frames))
+
+
+def animation_sound_pressure(omega, center, radius, amplitude, grid, frames,
+                             pulsate=False, figsize=(8, 8), **kwargs):
+    """Generate sound pressure animation."""
+    pressure = sfs.mono.source.pulsating_sphere(
+            omega, center, radius, amplitude, grid, inside=pulsate)
+    phasor = np.exp(1j * 2 * np.pi / frames)
+
+    fig = plt.figure(figsize=figsize)
+    ax = fig.gca()
+    im = sfs.plot.soundfield(np.real(pressure), grid, **kwargs)
+    ax.axis([grid[0].min(), grid[0].max(), grid[1].min(), grid[1].max()])
+    return animation.FuncAnimation(
+            fig, update_frame_pressure, frames,
+            fargs=(grid, pressure, radius, amplitude,
+                   phasor, pulsate, im, frames))
+
+
+if __name__ == '__main__':
+
+    # Pulsating sphere
+    center = [0, 0, 0]
+    radius = 0.25
+    f = 750  # frequency
+    omega = 2 * np.pi * f  # angular frequency
+
+    # Temporal sampling for animation
+    fs = f * 20  # sampling frequency
+    frames = int(np.round(fs / f))  # number of frames
+
+    # Axis limits
+    xmin, xmax = -1, 1
+    ymin, ymax = -1, 1
+
+    # Particle displacement
+    amplitude = 0.05  # amplitude of the surface displacement
+    grid = sfs.util.xyz_grid([xmin, xmax], [ymin, ymax], 0, spacing=0.025)
+    ani = animation_particle_displacement(
+            omega, center, radius, amplitude, grid, frames, c='Gray')
+    ani.save('pulsating_sphere_displacement.gif', fps=10, dpi=80,
```

Comment:
You should choose a different name than in the notebook (or the other way round), otherwise one will overwrite the other's result.